pkg/auth/api/signin.go

```diff
@@ -34,8 +34,9 @@ func (s *authService) SignIn(
 	if err != nil {
 		return nil, err
 	}
-	if request.Email != s.config.DemoSignInConfig.Email &&
```

Comment:
I fixed the condition because if one of the two was correct, it authorized and generated the token.